crates/edr_op/src/chains.rs

```diff
@@ -0,0 +1,80 @@
+use std::sync::LazyLock;
```

Comment:
For this, maybe split it up into:

hardfork
|
- op.rs
- base.rs

Given that these are essentially child-modules of `hardfork.rs`. Having that structure will also be easy if we adopt code generation in the future. One file can be generated per chain.